storybookjs/storybook
# Feat: Add support for Flow to @addons/storysource

Issue: Feat: #4955 

## What I did
Add `Prettier` Flow parser to `@addons/storysource`

`flow` is now an available option for parser under the `loader` options

## How to test
I added some flow type annotations to a few kitchen sink app stories to ensure that storybook was able to launch and storysource was able to parse Flow's syntax.

- Is this testable with Jest or Chromatic screenshots? 
Can create a mock file to test the inject decorator with flow stories

- Does this need a new example in the kitchen sink apps?
No

- Does this need an update to the documentation?
Yes, include flow as an available option.

If your answer is yes to any of these, please make sure to include it in your PR.
